website/src/content/getting-started/what-is-a-custom-application.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: What is a custom application?
+---
+
+<Subtitle>
+
+A custom application is a UI web application for the Merchant Center, designed to enable a project in the [commercetools platform](https://commercetools.com/) to tailor specific needs of a customer.
+
+</Subtitle>
+
+# About the Merchant Center
+
+The Merchant Center is an **administration tool to help customers manage the data** of their projects in the [commercetools platform](https://commercetools.com/).
+
+At its core, the Merchant Center is composed of multiple single-page web applications running behind a Proxy Router. You can read more about the architecture [here](/main-concepts/architecture).
+
+# Customizing the Merchant Center
+
+The Merchant Center provides a set of functionalities built-in and maintained by the commercetools platform team. For example managing a product catalog, orders, discounts etc.
+The built-in features are usually aligned with the functionalities of the [commercetools platform HTTP API](https://docs.commercetools.com/http-api.html).
+
+However, there are cases where a customer project requires extra functionalities that are specific to that customer needs. To solve some of these problem, commercetools allows to build custom applications for the Merchant Center.
```

Comment:
that customer needs => that customer's needs

"To solve some of these problem" => problem**s** plural.

"commercetools allows **you** to build custom applications for the Merchant Center."